schemas/ispyb/stored_programs/sp_insert_dc_plan.sql

```diff
@@ -0,0 +1,100 @@
+DELIMITER ;;
+CREATE OR REPLACE DEFINER=`ispyb_root`@`%` PROCEDURE `insert_dc_plan`(
+	OUT p_id int(11) unsigned,
+	p_diffactionPlanId int(10),
+	p_name varchar(20),
+	p_experimentKind enum('Default','MXPressE','MXPressO','MXPressE_SAD','MXScore','MXPressM','MAD','SAD','Fixed','Ligand binding','Refinement','OSC','MAD - Inverse Beam','SAD - Inverse Beam','MESH','XFE','Stepped transmission','XChem High Symmetry','XChem Low Symmetry','Commissioning'),
+	p_observedResolution float,
+	p_minimalResolution float,
+	p_exposuretime float,
+	p_oscillationRange float,
+	p_maximalResolution float,
+	p_screeningResolution float,
+	p_radiationSensitivity float,
+	p_anomalousScatterer varchar(255),
+	p_preferredBeamSizeX float,
+	p_preferredBeamSizeY float,
+	p_preferredBeamDiameter float,
+	p_comments varchar(1024),
+	p_DIFFRACTIONPLANUUID varchar(1000),
+	p_aimedCompleteness double,
+	p_aimedIOverSigmaAtHighestRes double,
+	p_aimedMultiplicity double,
+	p_aimedResolution double,
+	p_anomalousData tinyint(1),
+	p_complexity varchar(45),
+	p_estimateRadiationDamage tinyint(1),
+	p_forcedSpaceGroup varchar(45),
+	p_requiredCompleteness double,
+	p_requiredMultiplicity double,
+	p_requiredResolution double,
+	p_strategyOption varchar(200),
+	p_kappaStrategyOption varchar(45),
+	p_numberOfPositions int(11),
+	p_minDimAccrossSpindleAxis double,
+	p_maxDimAccrossSpindleAxis double,
+	p_radiationSensitivityBeta double,
+	p_radiationSensitivityGamma double,
+	p_minOscWidth float,
+	p_recordTimeStamp timestamp,
+	p_monochromator varchar(8),
+	p_energy float,
+	p_transmission float,
+	p_boxSizeX float,
+	p_boxSizeY float,
+	p_kappaStart float,
+	p_axisStart float,
+	p_axisRange float,
+	p_numberOfImages mediumint(9),
+	p_presetForProposalId int(10) unsigned,
+	p_beamLineName varchar(45),
+	p_detectorId int(11),
+	p_distance double,
+	p_orientation double,
+	p_monoBandwidth double,
+	p_centringMethod enum('xray','loop','diffraction','optical'),
+	p_userPath varchar(100),
+	p_robotPlateTemperature float,
+	p_exposureTemperature float,
+	p_experimentTypeId int(10) unsigned,
```

Comment:
GDA doesn't know about these foreign key values. I suggest removing this parameter, but populating the column by doing a lookup in the `ExperimentType` table. I'll explain per Slack/email how to do that ...